service/storage_service.cc

```diff
@@ -904,6 +904,12 @@ future<> storage_service::handle_state_normal(inet_address endpoint) {
         } else if (existing && *existing == endpoint) {
             tmptr->del_replacing_endpoint(endpoint);
         } else {
+            auto nodes = _gossiper.get_nodes_with_host_id(host_id);
+            bool left = std::any_of(nodes.begin(), nodes.end(), [this] (const gms::inet_address& node) { return _gossiper.is_left(node); });
+            if (left) {
+                slogger.info("Skip to set host_id={} to be owned by node={}, because the node is removed from the cluster", host_id, endpoint);
```

Comment:
Maybe we should mention the old IP of this node as well.

Reply:
I added the nodes used to own the host id in the log.